packages/web/src/store/sagas.ts

```diff
@@ -150,6 +152,7 @@ export default function* rootSaga() {
     artistRecommendationsSagas(),
     buyAudioSagas(),
     changePasswordSagas(),
+    chatWebSagas(),
```

Comment:
I'm not in love with this name, or the splitting out of the sagas into two separate sagas. Couldn't think of a better way to handle the redirect after creating a chat though, since common doesn't know about the routing per client